.github/workflows/ci-smart-contract.yml

```
@@ -0,0 +1,185 @@
+name: Smart Contract CI
+#  Smart Contract CI workflow runs CI test for checking if smart contract works well
+#  This workflow is run on pushes to main & every Pull Requests where a .go, .mod, .sum, ci_test/**.sh, and this file have been changed
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+env:
+  TEST_DOCKER_IMAGE: finschianode:test
```

I think using `test` is the common name. How about using the below:
```suggestion
  TEST_DOCKER_IMAGE: finschianode:smartcontractci
```